CDCgov/prime-fhir-converter
# Initial migration of FHIR to HL7 v2 library from ReportStream Repo

This PR copies the stable FHIR to HL7 v2 library code and files from Feb 2023 from the ReportStream repository (see [RS repository at this commit](https://github.com/CDCgov/prime-reportstream/tree/ce434e87556c2653a621bfbcb46a95c8fa89f897)). No code changes were made to the code.

Test Steps:
1. Checkout this branch.
2. Run `./gradlew build`.
3. Verify the code compiles, and that the unit tests run and pass.
4. Run `./gradlew ktlintCheck` and verify that the build passes.

## Changes
- Copied library code from RS repo
- Copied unit tests from RS repo
- Generated gradle build file
